Dockerfile.frontend

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine as build
```

Comment:
Is this is part of what we show to customers as an example? If so just wondering if it should use a RH container like ubi

Reply:
What's an RH container alternative that we could try?  Also, is there one for nginx that we could try?

Reply:
Oh sorry, misunderstood that.  Your above comment sounds like a plan 👍